apps/condo/domains/banking/schema/BankAccount.test.js

```diff
@@ -397,37 +397,21 @@ describe('BankAccount', () => {
             }, `Cannot connect to BankIntegrationContext, used by another BankAccount(id="${anotherBankAccount.id}")`)
         })
 
-        test('can\'t update connection to BankIntegrationContext', async () => {
+        test('cannot connect BankIntegrationContext if BankAccount is already connected to some integrationContext', async () => {
             const [organization] = await createTestOrganization(adminClient)
 
-            const [bankIntegrationContext1] = await createTestBankIntegrationContext(adminClient, bankIntegration, organization)
-            const [bankAccount1] = await createTestBankAccount(adminClient, organization, {
-                integrationContext: { connect: { id: bankIntegrationContext1.id } },
-            })
-
-            await catchErrorFrom(async () => {
-                await updateTestBankAccount(adminClient, bankAccount1.id, {
-                    integrationContext: { disconnect: { id: bankIntegrationContext1.id } },
-                })
-            }, (e) => {
-                expect(e.errors[0].message).toContain('Field "integrationContext" is not defined by type "BankAccountUpdateInput"')
-            })
+            const [bankIntegrationContext] = await createTestBankIntegrationContext(adminClient, bankIntegration, organization)
+            const [anotherIntegrationContext] = await createTestBankIntegrationContext(adminClient, bankIntegration, organization)
 
-            const [bankIntegrationContext2] = await createTestBankIntegrationContext(adminClient, bankIntegration, organization)
-            await createTestBankAccount(adminClient, organization, {
-                integrationContext: { connect: { id: bankIntegrationContext2.id } },
+            const [bankAccount] = await createTestBankAccount(adminClient, organization, {
+                integrationContext: { connect: { id: bankIntegrationContext.id } },
             })
 
-            await catchErrorFrom(async () => {
-                await updateTestBankAccount(adminClient, bankAccount1.id, {
-                    integrationContext: {
-                        disconnect: { id: bankIntegrationContext1.id },
-                        connect: { id: bankIntegrationContext2.id },
-                    },
+            await expectToThrowValidationFailureError(async () => {
+                await updateTestBankAccount(adminClient, bankAccount.id, {
+                    integrationContext: { connect: { id: anotherIntegrationContext.id } },
                 })
-            }, (e) => {
-                expect(e.errors[0].message).toContain('Field "integrationContext" is not defined by type "BankAccountUpdateInput"')
-            })
+            }, 'Integration reassignment is not allowed for BankAccount')
```

Comment:
Better to move error text to constant. You can change error text later = but the name of error constant will remain